credentialz/gnsi-credentialz.yang

```diff
@@ -108,6 +127,7 @@ module gnsi-ssh {
         description
             "A system role credentials freshness information.";
 
+        uses user-console-credentials-version;
         uses user-ssh-credentials-version;
     }
     deviation  "/oc-sys:system/oc-sys:aaa/oc-sys:authentication/oc-sys:users/" +
```

Comment:
This is a nice idea -- I like the idea that there is an explicit expectation set as to not allowing this via gNMI when gNSI is used.